Autopilot/TelemetryManager/Telemetry_Manager.c

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file Telemetry_Manager.c
+ * @author Jingting Liu
+ * @date January 14th, 2021
+ */
+
+#include <stdio.h>
+#include <common/mavlink.h>
+
+#include "telemetry_state_machine.h"
+
+
+
+TM_STATE_e current_state = INIT;
+
+// TODO: Add FreeRTOS specifics, return type, waits etc
+void telemetery_manager()
+{
+    while (1)
```

Comment:
No while(1)! The freeRtos stuff will live above your state machine and will control how it's called. Having 1 call execute 1 cycle of the state machine is sufficient.